coral/src/app/features/approvals/schemas/SchemaApprovals.tsx

```diff
@@ -68,7 +68,10 @@ function SchemaApprovals() {
       <option> three </option>
     </NativeSelect>,
 
-    <NativeSelect labelText={"Filter by cluster"} key={"filter-cluster"}>
+    <NativeSelect
+      labelText={"Filter by environment"}
```

Comment:
I think in the Harshini's feedback about casing, "Klaw specific entities" are capitalized. So, `={"Filter by Environment"}` is correct I believe.

Reply:
💡